plenum/common/constants.py

```diff
@@ -185,6 +185,9 @@
 
 TXN_AUTHOR_AGREEMENT_TEXT = "text"
 TXN_AUTHOR_AGREEMENT_VERSION = "version"
+TXN_AUTHOR_AGREEMENT_DIGEST = "digest"
+TXN_AUTHOR_AGREEMENT_RETIRED = "retired"
+TXN_AUTHOR_AGREEMENT_TIMESTAMP = "timestamp"
```

Comment:
Is there any reason why we call it abstract "timestamp" instead of more clear "creation time"?